### DISCUSSION TITLE
ERROR : Could not find nodes!

### DISCUSSION THREAD

**Discussion by UraniumCakes** [idaholab/moose#28371] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Hi everyone, I've been having a lot of trouble with MOOSE recently. I'm a beginner and I am trying to get MOOSE to read a file with a mesh but I keep getting an error saying that the nodes can't be found. However, I know that the unv file has nodes and a mesh so I don't understand why this is happening. I don't think it's an issue from my input file because my colleague can't seem to find anything wrong with it. Has this ever happened to anyone? If so how have you fixed the issue? Any help would be greatly appreciated, thank you.

### (Optional) code in question / simulation log / errors

```shell
*** ERROR ***
ERROR: Could not find nodes!
Stack frames: 18
0: libMesh::print_trace(std::ostream&)
1: libMesh::MacroFunctions::report_error(char const*, int, char const*, char const*, std::ostream&)
2: libMesh::UNVIO::read_implementation(std::istream&)
3: libMesh::UNVIO::read(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
4: libMesh::NameBasedIO::read(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
5: libMesh::UnstructuredMesh::read(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, void*, bool, bool)
6: FileMeshGenerator::generate()
7: MeshGenerator::generateInternal()
8: MeshGeneratorSystem::executeMeshGenerators()
9: Action::timedAct()
10: ActionWarehouse::executeActionsWithAction(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
11: ActionWarehouse::executeAllActions()
12: MooseApp::runInputFile()
13: MooseApp::run()
14: ./navier_stokes-opt(+0x30ea) [0x55667674a0ea]
15: main
16: __libc_start_main
17: ./navier_stokes-opt(+0x338b) [0x55667674a38b]
[0] ../src/mesh/unv_io.C, line 218, compiled Jul 19 2024 at 13:32:04
```

**Comment by GiudGiud** [comment]
Hello

The .unv file must not be read properly. Or it is nor formatted as expected.
How did you generate it?
Can Paraview read it?

**Reply by UraniumCakes** [reply]
```
[Mesh]
    [mesh]
      type = FileMeshGenerator
      file = 'HYLIFE_II_2D_medium.unv'
    []
     [merge]
       type = RenameBlockGenerator
       input = mesh
       old_block = 'Solid_1 Solid_2'
       new_block = '0 0'
      []
      construct_side_list_from_node_list=true        
```                                    
This is how I generated the mesh in moose, I created the mesh from salome though. I'm having a bit of trouble reading it in paraview because there are no local readers for a unv format.

**Reply by GiudGiud** [reply]
.unv is for 
I-deas Universal format

is that what salome is creating?

what is the list of options in salome?
all the ones we can read are there:
https://mooseframework.inl.gov/source/meshgenerators/FileMeshGenerator.html
